modules/datastore/datastore.install

```diff
@@ -21,3 +23,72 @@ function datastore_update_8001(&$sandbox) {
     $schema->addField($table, 'checksum', $spec);
   }
 }
+
+/**
+ * Create new "dkan_post_import_job_status" table.
+ */
+function datastore_update_9055(&$sandbox) {
```

Comment:
9055 is an interesting choice...

Reply:
This has been fixed. Thank you for catching that.